src/css/molecules/header-strip.scss

```diff
@@ -2,7 +2,36 @@
   background-color: $color-black;
   color: $color-white;
 
-  h1 {
-    margin: $spacing-sm 0 $spacing-md;
+  .icon-wrapper {
+    float: left;
+    fill: currentColor;
```

Comment:
Not sure of the code standard on this project and whether you are using Prettier but, generally it is common to have an empty line here

Reply:
Prettier checks everything via Therapist on commit, so it passed. However, i've added it in later commits to the same file (on another branch) 👍